src/cargo/ops/resolve.rs

```diff
@@ -147,8 +147,6 @@ pub fn resolve_ws_with_opts<'gctx>(
         // Second, resolve with precisely what we're doing. Filter out
         // transitive dependencies if necessary, specify features, handle
         // overrides, etc.
-        let _p = profile::start("resolving with overrides...");
```

Comment:
This is dropped and no profiling for the code path. Just pointing out but not too worried about the change.

Reply:
`add_overrides` has an `instrument` attribute.   The change in coverage is for the warning which I assume wasn't going to be too important.